app/templates/application.hbs

```diff
@@ -22,14 +22,19 @@
     searchTerms=searchTerms
     typeTitleLookup=(hash
       lot='Addresses'
-      waterfront-park-name='Park Name')
+      waterfront-park-name='Waterfront Space Profiles')
     onClear=(action (mut geocodedFeature null))
     class="search-container"
     as |search|}}
     {{#if (eq search.result.type 'lot')}}
-      <span class="icon tax-lot"></span> <span class="subdued">{{search.result.label}}</span>
+      {{fa-icon 'map-pin' fixedWidth=true}} <span class="dark-gray">{{search.result.label}}</span>
```

Comment:
```suggestion
      {{fa-icon 'map-pin' fixedWidth=true}}
      <span class="dark-gray">
        {{search.result.label}}
      </span>
```

Can we research how to format this better without the newlines wreaking havoc? I'll ask a few people in Discord. I know it requires the `~` which look bad, but maybe we can make them the default.

Reply:
New lines won't cause an issues here. But do we really want an inline element's tags to be on their own lines? 

(I am gonna change this tho. Both spans are the same, so there can just be one outside the `{{if}}`. The condition is just for the icon.)

Reply:
> But do we really want an inline element's tags to be on their own lines?

Yes, I should think so - it's way easier to read than inline, right? We should agree on a linter of some sort for templates. 

Let me know if I should look into the newline issue more. Thanks

Reply:
I like Google's rule for what HTML tags should be on their own lines: 

> Use a new line for every block, list, or table element, and indent every such child element.

https://google.github.io/styleguide/htmlcssguide.html#HTML_Formatting_Rules

I'd prefer inline tags not being on new lines. But we need wiggle room for situations where the code is hard to read. Linting HTML might get pretty finicky, tho it could be valuable to lint handlebars.